ddocs/medic/_attachments/translations/messages-en.properties

```
@@ -1385,6 +1387,32 @@ user.place.help = All users must be associated with a place in the system to be
 user.username.help = This is what you will use to log in to the app.
 username.invalid = Invalid user name. Valid characters are lower case letters, numbers, underscore (_), and hyphen (-).
 username.taken = Username "{{username}}" already taken.
+users.manage.back = Back to user list
+users.manage.add-single-user = Add User
+users.manage.import-users = Import from file
+users.import.title = Import from file
+users.import.instructions = To add multiple users, please upload the appropriate spreadsheet.
+users.import.accepted-types = Only .csv file are accepted.
+users.import.upload-button = Upload .csv
+users.import.confirm-upload-text = Are you sure you want to upload
+users.import.confirm-upload = Yes, upload
+users.import.cancel-upload = Cancel
+users.import.processing.title = Processing users
+users.import.processing.instructions = This process can take a long time, if you're uploading a lot of users, please be patient.
+users.import.unavailable.title = Currently unavailable
+users.import.unavailable.instructions = Someone else in the system is currently in the process of adding multiple users.
+users.import.unavailable.retry = Please try again later
+users.import.summary.added-users = users added
+users.import.summary.previously-added-users = previously imported
+users.import.summary.failed-users = contained errors
+users.import.summary.click = Click
+users.import.summary.here = here
+users.import.summary.download-status-file = to download your status file.
+users.import.summary.resubmit.instructions = To resubmit the users containing errors :
```

```suggestion
users.import.summary.resubmit.instructions = To resubmit the users containing errors:
```

I think it's like that